static/global.css

```diff
@@ -2,7 +2,32 @@
 
 @font-face {
   font-family: 'Visby CF';
-  src: url('fonts/VisbyCF-Bold.woff2');
+  font-display: block;
+  font-weight: 700;
+  src: url('fonts/VisbyCF-Bold.woff2') format('woff2'),
+    url('fonts/VisbyCF-Bold.woff') format('woff');
```

Comment:
Might I suggest converting this into multiple other formats (.eot, .otf, .svg, .ttf) for maximum compatibility?

I believe you can use [FontForge](https://fontforge.org/en-US/) to convert to most (all?) of these. `fontforge -lang=ff -c 'Open($1); Generate($2)' VisbyCF-Bold.woff VisbyCF-Bold.ttf`.

Reply:
Used this as a guide for browser compatibility: https://css-tricks.com/snippets/css/using-font-face/

We're already breaking layouts in IE by using CSS Grid, so not worried about using `.eot` or `.otf`. I'll generate a `ttf` though because why not? 🤷‍♀